docs/install-and-setup.md

```diff
@@ -276,3 +276,14 @@ source-bash $(jj util completion)
 ```
 
 (dynamic completions not available yet)
+
+### Powershell
+
+Insert this line in your $PROFILE file:
+(usually C:\Users\USER\Documents\PowerShell\Microsoft.PowerShell_profile.ps1)
```

Comment:
nit: would it be clearer to replace "USER" by "$USER" or "<users>"? (i definitely think it would for linux users, but i'm not sure about windows users)

Reply:
(I edited the message to treat the "&lt;user&gt;" as an html tag)

Reply:
might be clearer to have `%USER%` ?

Reply:
or even `(%HOME%\Documents\PowerShell\Microsoft.PowerShell_profile.ps1)`.  Not sure what a typical Windows user expect.

Reply:
Sure, I'll let you decide.